src/pages/Faq/FaqModal.js

```diff
@@ -0,0 +1,336 @@
+import React, { useState } from "react";
+import { RiDeleteBin5Line } from "react-icons/ri";
+import { AiOutlineClose } from "react-icons/ai";
+import styled from "styled-components";
+import theme from "styles/theme";
+
+const FaqModal = ({ handleModal }) => {
+  const [inputValue, setInputValue] = useState({
+    title: undefined,
+    content: undefined,
+    email: undefined,
+    phoneNumber: undefined,
+    check: false,
+  });
+  const [files, setFiles] = useState([]);
+
+  const handleSubmit = () => {
+    /* TODO: Post요청 */
+  };
+
+  const validate =
+    inputValue.title === undefined ||
+    inputValue.content === undefined ||
+    inputValue.email === undefined ||
+    inputValue.phoneNumber === undefined ||
+    inputValue.check === false;
+
+  const handleForm = (e) => {
+    const { name, value } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
```

Comment:
```suggestion
    setInputValue(prev => ({ ...prev, [name]: value }));
```
여기도 함수형 업데이트로 구현해주세요!